test/functional/cmdLineTests/CDSAdaptorTest/resources/TestBundle/src/com/ibm/testbundle/SomeMessageV1.java

```diff
@@ -1,17 +1,39 @@
-package com.ibm.testbundle;
-
-/**
- * A dummy program to print some message
- *  
- * @author administrator
- *
- */
-public class SomeMessageV1 {
-
-	/* If weaving hook bundle (com.ibm.weavinghooktest) is installed and running, 
-	 * it will replace the class bytes of this class with that of SomeMessageV2.
-	 */
-	public void printMessage() {
-		System.out.println("A message from original class");
-	}
-}
+/*******************************************************************************
+ * Copyright (c) 2001, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+package com.ibm.testbundle;
+
+/**
+ * A dummy program to print some message
+ *  
+ * @author administrator
```

Comment:
This author annotation can be removed.

Reply:
Author annotation have been removed. Thanks.